flocker/control/test/test_httpapi.py

```diff
@@ -42,6 +45,7 @@
 from .._clusterstate import ClusterStateService
 from .._config import (
     FlockerConfiguration, FigConfiguration, model_from_configuration)
+from .._model import Leases
```

Comment:
Minor quibble, but the rest of our models used in these tests are imported from module level defined in `__init__.py`